.github/workflows/ci.yml

```diff
@@ -34,14 +34,17 @@ jobs:
 
     - name: Build (Release)
       run: |
-        dotnet build DprintPluginRoslyn -c Release --runtime win-x64
-        dotnet build DprintPluginRoslyn -c Release --runtime linux-x64
-        dotnet build DprintPluginRoslyn -c Release --runtime osx-x64
+        dotnet build DprintPluginRoslyn -c Release --self-contained --runtime win-x64
+        dotnet build DprintPluginRoslyn -c Release --self-contained --runtime linux-x64
+        dotnet build DprintPluginRoslyn -c Release --self-contained --runtime osx-x64
+        dotnet build DprintPluginRoslyn -c Release --self-contained --runtime osx.12-arm64
 
     - name: Package
       run: |
         cd DprintPluginRoslyn/bin/Release/net6.0/osx-x64
         zip -r ../../../../../dprint-plugin-roslyn-x86_64-apple-darwin.zip ./*
+        cd DprintPluginRoslyn/bin/Release/net6.0/osx.12-arm64
```

Comment:
Oh, should be:

```suggestion
        cd ../osx.12-arm64
```

Reply:
You know, it probably shouldn't be doing `cd` in any of these. Not sure what I was thinking originally